admin/class-admin-asset-manager.php

```diff
@@ -298,17 +259,13 @@ protected function scripts_to_be_registered() {
 					self::PREFIX . 'components',
 				),
 			),
-			array(
-				'name' => 'wp-globals-backport',
-				'src'  => 'wp-seo-wp-globals-backport-' . $flat_version,
-				'deps' => $backport_wp_dependencies,
-			),
 			array(
 				'name' => 'yoast-modal',
 				'src'  => 'wp-seo-modal-' . $flat_version,
 				'deps' => array(
 					'jquery',
-					self::PREFIX . 'wp-globals-backport',
+					'wp-element',
```

Comment:
Shouldn't the ones where you remove `wp-global-backport` now have `react-dependencies` as a dependency? Given that that is essentially the `commons` file?

Reply:
The globals was previously built without the common chunks plugin and as such any files only depending on that shouldn't require the commons.